buildSrc/esh-version.properties

```diff
@@ -1,4 +1,4 @@
 eshadoop        = 8.0.0
 elasticsearch   = 8.0.0
 lucene          = 8.1.0-snapshot-e460356abe
-build-tools     = 7.1.1
\ No newline at end of file
+build-tools     = 7.4.0-SNAPSHOT
```

Comment:
Thanks for bumping this up. We're pretty close to being able to run against the master build tools here soon enough.